### PR TITLE
Some small changes to profile related things.

### PR DESCRIPTION
**Pull request by albertrdixon**
@lumoslabs/devops 
@srhaber

### CODE REVIEW

File: ssh.sh [hunk 1]
```diff
@@ -21,6 +21,29 @@ if [ ! -f ~/.bash_profile_includes/ssh-agent.sh ]; then
...
+                  cat ~/.ssh/config | \
+                          grep "^Host " | \
+                          awk '{print $2}'
+                  `
+          COMPREPLY=( $(compgen -W "${comp_ssh_hosts}" -- $cur))
+          return 0
+  }
+  complete -F _complete_ssh_hosts ssh
+EOF
+fi
```
Comment (srhaber) [inline ssh.sh]: Where did you pull this from?

File: ssh.sh [hunk 2]
```diff
@@ -21,6 +21,29 @@ if [ ! -f ~/.bash_profile_includes/ssh-agent.sh ]; then
...
+  cat >~/.bash_profile_includes/ssh-completion.sh <<-'EOF'
+  _complete_ssh_hosts ()
+  {
+          COMPREPLY=()
+          cur="${COMP_WORDS[COMP_CWORD]}"
+          comp_ssh_hosts=`cat ~/.ssh/known_hosts | \
+                          cut -f 1 -d ' ' | \
+                          sed -e s/,.*//g | \
+                          grep -v ^# | \
+                          uniq | \
```
Comment (srhaber) [inline ssh.sh]: Might need a `sort` before the `uniq` here?

Comment (albertrdixon) [inline ssh.sh]: good call, makes reading easier for sure

File: ssh.sh (hunk 1 above)
Comment (albertrdixon) [inline ssh.sh]: found a version of this online and have been using it for ssh tab completion for a bit

File: README.md [hunk 3]
```diff
@@ -2,9 +2,9 @@
...
 A simple way to get your laptop up and running
 
-## Get Going
+## Get Started
 
-In your terminal, simply paste this command and hit enter:
+In your terminal, simply paste this command and hit __Enter__:
 
 ```
 curl -sSL https://goo.gl/WQjlg0 | bash
```
Comment (matl33t) [inline README.md]: if we're going to use curl | bash, can we at least not use a link shortener?

Comment (jdoconnor) [inline README.md]: ha! i said something similar (out loud)

**Pull request merged**